tony-core/src/main/java/com/linkedin/tony/TonyApplicationMaster.java

```diff
@@ -764,6 +769,13 @@ public String registerWorkerSpec(String taskId, String spec) throws IOException
         LOG.info("Received cluster spec registration request from task " + taskId + " with spec: " + spec);
         task.setHostPort(spec);
         registeredTasks.add(taskId);
+
+        // Use chief worker as coordinator.
+        if (taskId.equals(COORDINATOR_ID) && framework == MLFramework.PYTORCH) {
+          // Hard coded to use tcp:// as backend.
```

Comment:
Can we add a TODO to support other backends like Gloo, MPI, and NCCL? Gloo and NCCL will allow us to take advantage of GPU features like direct memory access/write while TCP will not.